src/main/java/org/highmed/numportal/service/AqlService.java

```diff
@@ -119,7 +119,7 @@ public Page<Aql> getVisibleAqls(String loggedInUserId, Pageable pageable, Search
     AqlSpecification aqlSpecification = AqlSpecification.builder()
             .filter(searchCriteria.getFilter())
             .loggedInUserId(loggedInUserId)
-            .loggedInUserOrganizationId(userDetails.getOrganization().getId())
+            .loggedInUserOrganizationId((userDetails.getOrganization() != null) ? userDetails.getOrganization().getId() : null)
```

Comment:
This case cant happen, because you cant registrated an user without organistation because you need a whitelisted mail-surfix.

Reply:
This is true; but it can happen when either the white-listing is deactivated, or for admin users that don't have an organisation